app/__main__.py

```diff
@@ -24,28 +25,39 @@ async def setup_hook(self) -> None:
         await self.tree.sync()
 
 
+client = DiscordClient(intents=discord.Intents.default())
+database = Database()
```

Comment:
Can you store `database` as an attribute on `DiscordClient`? Should just require overwriting `__init__`.

This isn't necessary ATM but will be necessary when we need to be in an async context to make the database.